src/Microsoft.SqlTools.ServiceLayer/Agent/Contracts/AgentJobInfo.cs

```diff
@@ -61,4 +61,16 @@ public class AgentJobInfo
         public AgentScheduleInfo[] JobSchedules { get; set; }
         public AgentAlertInfo[] Alerts { get; set; }
     }
+
+    /// <summary>
+    /// a class for storing variour properties of notebook Jobs
+    /// </summary>
+    public class AgentNotebookInfo : AgentJobInfo
+    {
+        public string TemplateId { get; set; }
```

Comment:
consider adding comments of these properties describing they are..specifically the distinction between Target and Execution databases, etc.